### PR TITLE
📖 Fix anchor err in 20190610-machine-states-preboot-bootstrapping.md

### PR DESCRIPTION
**Pull request by chymy**
**What this PR does / why we need it**:
Fix anchor err and  remove the TOC from the document metadata (L3-L72) in 20190610-machine-states-preboot-bootstrapping.md
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: docs/proposals/20190610-machine-states-preboot-bootstrapping.md [hunk 1]
```diff
@@ -122,13 +121,13 @@ status: implemented
...
-      * [As an infrastructure provider consumer, I would like to have validation for the provider\-specific data I need to give the system to have it provision a machine\.](#as-an-infrastructure-provider-consumer-i-would-like-to-have-validation-for-the-provider-specific-data-i-need-to-give-the-system-to-have-it-provision-a-machine)
-      * [As an infrastructure  provider author, I would like to build a controller to manage provisioning machines using tools of my own choosing\.](#as-an-infrastructure--provider-author-i-would-like-to-build-a-controller-to-manage-provisioning-machines-using-tools-of-my-own-choosing)
-      * [As an infrastructure provider author, I would like to build a controller to manage provisioning machines without being restricted to a CRUD API\.](#as-an-infrastructure-provider-author-i-would-like-to-build-a-controller-to-manage-provisioning-machines-without-being-restricted-to-a-crud-api)
+      * [As a Kubernetes operator, I’d like to provide custom bootstrap data without the use of a Kubernetes controller.](#as-a-kubernetes-operator-id-like-to-provide-custom-bootstrap-data-without-the-use-of-a-kubernetes-controller)
```
Comment (fabriziopandini) [inline docs/proposals/20190610-machine-states-preboot-bootstrapping.md:124]: TBH I'm for removing single users stories from the TOC given that they are single sentences and not paragraphs (Let's change the text style from h4 to normal/bullet list)

**Review by fabriziopandini — COMMENTED** [review]
On small nit;
Also, given that you are working on this doc, could you kindly remove the TOC from the document metadata (L3-L72)?

File: docs/proposals/20190610-machine-states-preboot-bootstrapping.md (hunk 1 above)
Comment (chymy) [reply docs/proposals/20190610-machine-states-preboot-bootstrapping.md]: Thanks @fabriziopandini, done.

**Review by fabriziopandini — COMMENTED** [review]
/lgtm

**Review by vincepri — APPROVED** [review]
/approve

**Pull request merged**